ProteGO/Common/Bluetooth/Scanner/DeviceEvent.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import CoreBluetooth
+
+/// Device events emitted by scanner, peripheral or central manager.
+enum DeviceEvent {
```

Comment:
Czy to przypadkiem są nie tylko eventy a także polecenia do wykonania na urządzeniu?

Reply:
Polecenia są zwarte w `DeviceEffect`

Reply:
To może zmienić `StartSynchronization` co brzmi jak polecenie na `SynchronizationStarted` co brzmi jak raport stanu?

Reply:
👍